v2/samples/direct_method_invocation_sample/EngineSimulatorModule/.gitignore

```diff
@@ -0,0 +1,31 @@
+# .NET Core
```

Comment:
Can you rename the project to EngineSimulator so it is consist with the docker image name?